File: Arrays/Java/Count_Odd_Even.java
class Solution
{
    public void countOddEven(int[] arr, int n)
    {
        // Code here
        int even = 0;
        int odd = 0;
        for (int i=0; i<n; i++) {
            if (arr[i]%2==0) {
                even += 1;
            }
            else {
                odd += 1;
            }
        }
       System.out.println(odd + " " + even);
    }
}

File: Solutions/Python/Count_Odd_Even.py
class Solution:
	def countOddEven(self, arr, n):
		#Code here
		even = 0
		odd = 0
		for i in arr:
		    if i%2==0:
		        even += 1
		    else:
		        odd += 1
		print(odd, even)
